checkov/runner_filter.py

```diff
@@ -34,7 +34,8 @@ def __init__(
             excluded_paths: Optional[List[str]] = None,
             all_external: bool = False,
             var_files: Optional[List[str]] = None,
-            skip_cve_package: Optional[List[str]] = None
+            skip_cve_package: Optional[List[str]] = None,
+            filtered_policy_ids: List[str] = []
```

Comment:
also here, please don't use `[]` as a default, instead use `None` and then check if it is `None` and set `[]` for the variable.